ingest-app/src/cmr/ingest/services/granule_bulk_update_service.clj

```diff
@@ -250,6 +278,22 @@
           (assoc :revision-date (time-keeper/now))
           (assoc :user-id user-id)))))
 
+(defmethod update-granule-concept :append_to_field:s3link
```

Comment:
The only thing different in this function from the `:update_field:s3link` function is the call to `append-s3-url`. There is no need to copy paste all these code in this function. Refactor it to be a function and call it from both `update-granule-concept :append_to_field:s3link` and `update-granule-concept :update_field:s3link`.